### PR TITLE
fixed wrong doc link

### PR DESCRIPTION
**Pull request by Yanni8**
This link points to [nothing](https://microsoft.github.io/autogen/docs/Use-Cases/FAQ#set-your-api-endpoints). I think the right link should point to [this](https://microsoft.github.io/autogen/docs/FAQ#set-your-api-endpoints)

### CODE REVIEW

**Comment by jackgerrits** [comment]
I just looked on the deployed version of the website and it seems like the link is working (https://microsoft.github.io/autogen/docs/Use-Cases/agent_chat). In what environment did you notice the broken link?

**Comment by Yanni8** [comment]
Hey @jackgerrits

> I just looked on the deployed version of the website and it seems like the link is working (https://microsoft.github.io/autogen/docs/Use-Cases/agent_chat). In what environment did you notice the broken link?

Well I think I found the issue. The link on the website redirects me to https://microsoft.github.io/autogen/docs/Use-Cases/agent_chat

Which is greate. But directly requesting the page https://microsoft.github.io/autogen/docs/Use-Cases/agent_chat
 redirects me to -> https://microsoft.github.io/autogen/docs/Use-Cases/agent_chat/ (There is an aditional `/`) which means that the link won't work.

Does this also happen to you?

**Comment by jackgerrits** [comment]
Ah thanks for explaining the issue! If you reference the `FAQ.md` file directly and let docusaurus generate the link then I think things work correctly.

File: website/docs/Use-Cases/agent_chat.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ or Pydantic models:
 
 The following examples illustrates the process of registering a custom function for currency exchange calculation that uses type hints and standard Python datatypes:
 
-1. First, we import necessary libraries and configure models using [`autogen.config_list_from_json`](../FAQ#set-your-api-endpoints) function:
+1. First, we import necessary libraries and configure models using [`autogen.config_list_from_json`](../../FAQ#set-your-api-endpoints) function:
```
Comment (jackgerrits) [inline website/docs/Use-Cases/agent_chat.md:57]: I believe this is the correct fix

```suggestion
1. First, we import necessary libraries and configure models using [`autogen.config_list_from_json`](../FAQ.md#set-your-api-endpoints) function:
```

**Review by jackgerrits — APPROVED** [review]

**Comment by jackgerrits** [comment]
Thanks for finding this, and pushing through this fix!

File: website/docs/Use-Cases/agent_chat.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ or Pydantic models:
 
 The following examples illustrates the process of registering a custom function for currency exchange calculation that uses type hints and standard Python datatypes:
 
-1. First, we import necessary libraries and configure models using [`autogen.config_list_from_json`](../FAQ#set-your-api-endpoints) function:
+1. First, we import necessary libraries and configure models using [`autogen.config_list_from_json`](../FAQ.md#set-your-api-endpoints) function:
```
Comment (sonichi) [inline website/docs/Use-Cases/agent_chat.md:57]:
```suggestion
1. First, we import necessary libraries and configure models using [`autogen.config_list_from_json`](/docs/FAQ#set-your-api-endpoints) function:
```

Comment (jackgerrits) [reply website/docs/Use-Cases/agent_chat.md:57]: That also works

**Pull request merged**